<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>472</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="centralWidgetHLayout">
    <item>
     <widget class="QTabWidget" name="leftTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pointsTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <attribute name="title">
        <string>Точки</string>
       </attribute>
       <layout class="QVBoxLayout" name="pointsTabVLayout">
        <item>
         <widget class="QScrollArea" name="pointsScrollArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="pointsAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>300</width>
             <height>292</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="pointsVLayout">
            <property name="leftMargin">
             <number>4</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setupPathPointBtn">
          <property name="text">
           <string>Задать путь</string>
          </property>
         </widget>
        </item>
		<item>
		<widget class="QPushButton" name="addReferenceViewBtn">
          <property name="text">
           <string>Добавить опорный план</string>
          </property>
         </widget>
		</item>
        <item>
         <widget class="QPushButton" name="startMoveBtn">
          <property name="text">
           <string>Начать движение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopMoveBtn">
          <property name="text">
           <string>Отменить движение</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="angleTab">
       <attribute name="title">
        <string>Ракурс</string>
       </attribute>
       <layout class="QVBoxLayout" name="angleTabVLayout">
        <item>
         <widget class="QScrollArea" name="anglesScrollArea">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="anglesScrollAreaContent">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>300</width>
             <height>350</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="anglesVLayout"/>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addAngleBtn">
          <property name="text">
           <string>Добавить ракурс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteAngleBtn">
          <property name="text">
           <string>Удалить ракурс</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="rightTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>454</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="mapTab">
       <attribute name="title">
        <string>Карта</string>
       </attribute>
       <layout class="QVBoxLayout" name="mapTabVLayout">
		<item>
		    <widget class="QGroupBox" name="mapGroupBox">
			  <property name="sizePolicy">
			   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
				<horstretch>1</horstretch>
				<verstretch>0</verstretch>
			   </sizepolicy>
			  </property>
			  <layout class="QHBoxLayout" name="horizontalLayout_4">
			    <item>
         <widget class="QGroupBox" name="mapGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Выбор проекции карты</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="orthoProjectionBtn">
             <property name="text">
              <string>2-д карта</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="perspectiveProjectionBtn">
             <property name="text">
              <string>3-д карта</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="mapGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Действия с картой</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="loadMapBtn">
             <property name="text">
              <string>Загрузить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteMapBtn">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
			  </layout>
			 </widget>
		</item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../test.qrc"/>
 </resources>
 <connections/>
</ui>
